packages/types/src/dialog.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
Nice that we'll just be able to poof this file when major version bump comes along 🧹 !